client/config/toml.go

```diff
@@ -63,7 +63,6 @@ func getClientConfig(configPath string, v *viper.Viper) (*ClientConfig, error) {
 	v.AddConfigPath(configPath)
 	v.SetConfigName("client")
 	v.SetConfigType("toml")
-	v.AutomaticEnv()
```

Comment:
In this PR we focus only on flags and client config. We keep env variables out of scope due to simplicity sake. We will address env variable issue in #8179